bouncer/src/repo/core/model/bson/repo_node_mesh.cpp

```diff
@@ -265,6 +265,17 @@ std::vector<repo_color4d_t> MeshNode::getColors() const
 	return colors;
 }
 
+std::vector<float> MeshNode::getIds() const
```

Comment:
as above, perhaps we should rename to getSubmeshIds

Reply:
Done!